workflow.rst

```diff
@@ -630,8 +612,8 @@ requires:
                         <framework:from>reviewed</framework:from>
                         <framework:to>published</framework:to>
                         <framework:metadata>
-                            <framework:hour_limit>20</framework:priority>
-                            <framework:explanation>You can not publish after 8 PM.</framework:priority>
+                            <framework:hour_limit>20</framework:hour_limit>
```

Comment:
Also, I check this i made mistake documenting metadata for xml, see here
https://github.com/symfony/symfony-docs/pull/11580/files#diff-c3ca19ec1845c786182534bf1afe2414R615
this change should go for 4.2 as well

Reply:
As `4.2` is end of maintenance now, lets ignore it for this branch!